src/api/l_graphics.c

```diff
@@ -721,7 +721,7 @@ static int l_lovrGraphicsGetBuffer(lua_State* L) {
     default: {
       Blob* blob = luax_totype(L, 1, Blob);
       if (blob) {
-        info.length = blob->size / info.stride;
+        info.length = (graphics_size)blob->size / info.stride;
```

Comment:
I don't think `graphics_size` is the correct type to use for a buffer length.  A buffer *size*, the size of its memory in bytes, should be a graphics_size, but the length is not a memory size and `uint32_t` would be more appropriate.